source/renderer/app/components/widgets/forms/InlineEditingInput.js

```diff
@@ -301,11 +301,9 @@ export default class InlineEditingInput extends Component<Props, State> {
           type="text"
           maxLength={maxLength}
           label={label}
-          onFocus={this.onFocus}
```

Comment:
@aleksandardjordjeviciohk please revert this

Reply:
Done